src/main/java/com/spikes2212/command/drivetrains/commands/DriveTank.java

```diff
@@ -17,26 +19,12 @@
 
     /**
      * This constructs a new {@link DriveTank} command that moves the given
-     * {@link TankDrivetrain} according to constant left side and right side speeds.<br>
-     * Positive values move forwards.
-     *
-     * @param drivetrain the tank drivetrain this command operates on.
-     * @param leftSpeed  the speed to move the left side with.
-     * @param rightSpeed the speed to move the right side with.
-     */
-
-
-    /**
-     * This constructs a new {@link DriveTank} command that moves the given
-     * {@link TankDrivetrain} acording to speed values from Double
-     * {@link Supplier}s for left and right sides.<br>
+     * {@link TankDrivetrain} according to speed values from Double {@link Supplier}s for left and right sides.<br>
```

Comment:
where the `<br>` start and end?

Reply:
<br> is a single tag, it starts and ends in the same tag ( no need for another tag to close him )